### PR TITLE
Updated daily spending wallet landing page

### PR DESCRIPTION
**Pull request by sbddesign**
For #782. Work in-progress.

TODO: Get a finished figma prototype for the "see it in action section".

💻 [Deploy Preview](https://deploy-preview-904--bitcoin-design-site.netlify.app/guide/daily-spending-wallet/)

### CODE REVIEW

**Comment by sbddesign** [comment]
Regarding @Bosch-0's comments on yesterday's call about making the chapters list more visual, here are some ideas.

ChaptersVisual2

ChaptersVisual1

I prefer the second one. If we use the page heading visuals, they will not carry over well at a small size. Let's just use an icon to represent each idea and a color.

I think it's a good idea to keep the content in a somewhat linear fashion. While they can also be read and appreciated independently, they do work together to tell a story about how this product works. I think this makes more sense than a very visual grid layout.

**Comment by Bosch-0** [comment]
I like the second one more also, could make the images radius slightly rounded to match other areas of the site (such as the home page below). 

image

File: guide/daily-spending-wallet/landing-page.md [hunk 1]
```diff
@@ -30,71 +30,189 @@ Illustration source:
...
+   </div>
+</div>
+
+<div class="center product-feature" markdown="1">
+   {% include picture.html
+      image = "/assets/images/guide/daily-spending-wallet/private-key.png"
+      retina = "/assets/images/guide/daily-spending-wallet/private-key@2x.png"
+      mobile = "/assets/images/guide/daily-spending-wallet/private-key-mobile.png"
+      mobileRetina = "/assets/images/guide/daily-spending-wallet/private-key-mobile@2x.png"
+      alt-text = "Graphic of a key"
```
Comment (GBKS) [inline guide/daily-spending-wallet/landing-page.md:58]: No need to include the word "Graphic" in alt-text. Overall, you may not even need alt-text for these 4 graphics, they don't really add much information that is not already captured in the copy right next to them.

**Comment by GBKS** [comment]
The page looks great. Something I'd love to see are more cross-links in the top-half. We can link to the [day-to-day spending paragraph](https://deploy-preview-904--bitcoin-design-site.netlify.app/guide/designing-products/personal-finance/#day-to-day-spending) in the personal finance page, the [self-custody principle](https://deploy-preview-904--bitcoin-design-site.netlify.app/guide/getting-started/principles/#self-custody), [lightning services](https://deploy-preview-904--bitcoin-design-site.netlify.app/guide/how-it-works/lightning-services/), etc. So if anyone arrives at this page and is not familiar with some of those aspects, they can easily catch up.

As for "making things more visual", I think this is not a worthwhile goal without the addition of "in order to communicate more effectively". The small icons can work as visual shortcuts. For example, the cloud next to the words "Backup & recovery" very effectively tells you something extra about the content. The colors used do not match the strong purple in the header and prototype images, maybe those could be tweaks.

By the way, we already have two styles of boxes, on the [calendar](https://bitcoin.design/calendar/) and [projects](https://bitcoin.design/projects/) pages. Boxes are helpful when you have content that is visually somewhat unstructured. I also find that they are over-used and it's better to first try to resolve lack of visual structure by adjusting spacing, proportions, typography, etc. Just a random comment.

As for rounded corners, the original design direction from two years ago asked for sharp edges, which you can see in the next/previous buttons and footer. But TBH, who cares if they are slightly rounded in a few places. Part of the original direction was also for the style to be a bit of a neutral framework, so creatives have freedom of expression and don't get overly concerned about 100% consistency.

File: guide/daily-spending-wallet/landing-page.md (hunk 1 above)
Comment (sbddesign) [reply guide/daily-spending-wallet/landing-page.md]: Done

**Review by Bosch-0 — APPROVED** [review]

**Comment by sbddesign** [comment]
@GBKS Updated

**Review by Bosch-0 — APPROVED** [review]

**Pull request merged**